tests/cpu_bugs/meltdown.pm

```diff
@@ -0,0 +1,86 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: CPU BUGS on Linux kernel check
+# Maintainer: James Wang <jnwang@suse.com>
+
+package meltdown;
+use strict;
+use warnings;
+
+use base "consoletest";
+use base "Mitigation";
```

Comment:
No idea use base "Mitigation", since i have not find any other case use base this package.

Reply:
I need to override its function.